jenkins_pipelines/environments/common/pipeline-build-validation.groovy

```diff
@@ -472,7 +472,9 @@ def clientTestingStages() {
                     }
                     randomWait()
                     echo 'Bootstrap clients'
-                    res_init_clients = sh(script: "./terracumber-cli ${common_params} --logfile ${resultdirbuild}/testsuite.log --runstep cucumber --cucumber-cmd 'unset ${temporaryList.join(' ')}; ${env.exports} cd /root/spacewalk/testsuite; rake cucumber:build_validation_init_client_${node}'", returnStatus: true)
+                    // Increase by 1.6 the default timeout (800 seconds) to manage the events taking up to 750 seconds during bootstrap
+                    def temporary_timeout = env.default_timeout * 1.6
```

Comment:
What do you think if we define a `env.bootstrap_client_timeout = 800` just below `env.default_timeout = 500` and use it here?

Otherwise this calculation becomes a [magic number](https://refactoring.guru/es/replace-magic-number-with-symbolic-constant).

Reply:
Yes it's probably better. If I remember correctly, we had this default timeout so high for bootstrap reason. We may even be able to reduce a bit the max timeout if we separate it from the bootstrap features. But let's focus on getting BV stable first